src/components/Navbar.js

```diff
@@ -3,8 +3,9 @@ import { Link } from 'react-router-dom';
 
 const Navbar = () => (
   <>
-    <nav>
-      <ul>
+    <nav className="head">
+      <h1>Bookstore CMS</h1>
+      <ul className="links">
         <li><Link to="/">Books</Link></li>
         <li><Link to="/categories">Categories</Link></li>
       </ul>
```

Comment:
- [ ] Kindly, compare your implementation with the design provided to meet all the requirements

Original design

![header--1](https://user-images.githubusercontent.com/55635977/150628629-aecfbbcb-5cb2-4323-8b34-cc97dc7c2ebe.png)

Your design

![header--3](https://user-images.githubusercontent.com/55635977/150628631-d07a78d5-f66d-4f4e-871d-39d96bedcd95.png)